src/index.js

```diff
@@ -6,13 +6,13 @@ import * as serviceWorker from './serviceWorker';
 import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 import loginReducer from './reducers/login-reducer'
+import { composeWithDevTools } from 'redux-devtools-extension';
 
-const initailState = {
-  users:[],
-  isAuthorizated: false
-}
+const store = createStore(loginReducer, composeWithDevTools());
 
-const store = createStore(loginReducer, initailState);
+store.subscribe(() => {
+  console.log(store.getState());
```

Comment:
why do you need `console.log`

Reply:
My bad, i need it for debugging, of course isn’t for production